_config.yml

```diff
@@ -114,5 +114,7 @@ exclude:
   - gulpfile.js
   - circle.yml
   - css
+  - pa11y-results.json
 
 env: dev
+url: http://localhost:4000
```

Comment:
Add a `url` key to assure that the URLs in any local sitemap output properly. This key in redefined in production by Federalist to assure proper URLs for the final sitemap.